test-plone-4.2.x.cfg

```diff
@@ -17,3 +17,12 @@ defaults = ['-s', '${buildout:package-name}', '--auto-color', '--auto-progress']
 eggs =
     ${buildout:package-name} ${buildout:package-extras}
     ${buildout:test-eggs}
+
+[versions]
+
+# The lates version of plone isn't available in launchpad
+# https://launchpad.net/plone/4.2/4.2.7
+Plone = 4.2.6
```

Comment:
the plone-4.2.6.cfg does not exists https://raw.github.com/collective/buildout.plonetest/master/plone-4.2.6.cfg